support/cas-server-support-oauth/src/test/java/org/apereo/cas/ticket/refreshtoken/OAuthRefreshTokenExpirationPolicyTest.java

```diff
@@ -0,0 +1,52 @@
+package org.apereo.cas.ticket.refreshtoken;
+
+import org.apereo.cas.authentication.Authentication;
+import org.apereo.cas.authentication.principal.Service;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.Assert.*;
+
+/**
+ * Adding test to check expired refresh token.
+ *
+ * @author Phillip Rower
+ * @since 5.0.0
+ */
+public class OAuthRefreshTokenExpirationPolicyTest {
```

Comment:
Please rename the class to end in `Tests` (plural). Otherwise, it will never run as part of the build.